Comments/components/commentBrowser.js

```diff
@@ -42,12 +45,22 @@ export class CommentBrowser extends Window {
       this.initializeContainers();
       this.initializeExtents();
       this.relayoutWindow();
-      $world.addMorph(this);
+
       instance = this;
+      this.name = 'comment browser';
+      this.commentGroups = {}; // dict Morph id -> Comment group morph
+
+      this.makeVisible();
```

Comment:
War `show` schon belegt?

Reply:
Ich würde erwarten, dass ich `makeVisible`/`show` nun immer aufrufen kann, um den Browser sichtbar zu machen.
Wie wäre `addToWorld` oder so?
Braucht es die gesonderte Funktion? Anscheinend wird sie nur an dieser Stelle verwendet und trägt nicht unbedingt zum Verständnis bei

Reply:
`show` display a red frame around a morph

Reply:
`makeVisible` is another abstraction, as we tried different approaches to hiding and showing the browser. You can use makeVisible to show the browser at anytime.